podvm-builder/aws-podvm-image-cm.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: aws-podvm-image-cm
+  namespace: openshift-sandboxed-containers-operator
+data:
+  # PodVM image distro
+  PODVM_DISTRO: rhel
+
+  # Image
+  AMI_BASE_NAME: "podvm-image"
+  AMI_VERSION_MAJ_MIN: "0.0"
+  AMI_VOLUME_SIZE: "30"
+
+  # Packer Instance type
+  INSTANCE_TYPE: "c4.xlarge"
+
+  # Pod VM sources
+  CAA_SRC: "https://github.com/bpradipt/cloud-api-adaptor.git"
+  CAA_REF: "gpu-test"
+
+  # Booleans
+  INSTALL_PACKAGES: "no"
+  DOWNLOAD_SOURCES: "no"
+  CONFIDENTIAL_COMPUTE_ENABLED: "no"
+  DISABLE_CLOUD_CONFIG: "true"
```

Comment:
This may conflict with the peer-pods-cm value

Reply:
Actually for latest upstream this is no longer used with CAA (ie in peer-pods-cm). So I might have to have a conditional. I'll explore..